static/src/javascripts/projects/common/modules/experiments/tests/membership-message.js

```diff
@@ -20,23 +20,24 @@ define([
 
         this.id = 'MembershipMessage';
         this.start = '2015-06-17';
-        this.expiry = '2015-06-26';
+        this.expiry = '2015-07-14';
         this.author = 'David Rapson';
         this.description = 'Test if loyal users are encouraged to join Membership as a Supporter';
         this.audience = 1;
         this.audienceOffset = 0;
         this.successMeasure = 'Loyal users will be interested in becoming a Supporter';
-        this.audienceCriteria = 'Users who have seen at least 10 pages, message shown only on article pages';
+        this.audienceCriteria = 'Users who have seen 10 or more pages, message shown only on article pages';
         this.dataLinkNames = 'supporter message, hide, read more';
         this.idealOutcome = 'Users will sign up as a Supporter';
 
         this.canRun = function () {
             /**
              * Exclude adblock users to avoid conflicts with similar adblock Supporter message,
-             * only show to users who have viewed at least 10 pages.
+             * only show to users who have viewed 10 or more pages.
              */
             var alreadyVisited = storage.local.get('alreadyVisited') || 0;
-            return !detect.adblockInUse &&
+            return !detect.adblockInUse() &&
+                detect.getBreakpoint() !== 'mobile' &&
```

Comment:
We've only had feedback about it being obtrusive on small screens, I'm happy to keep it on tablet for now.